zerver/lib/narrow.py

```diff
@@ -919,6 +927,30 @@ def get_channel_from_narrow_access_unchecked(
     return None
 
 
+def update_narrow_terms_containing_with_operator(
+    realm: Realm, narrow: OptionalNarrowListT
+) -> OptionalNarrowListT:
+    if narrow is not None:
```

Comment:
You can use the early-return pattern to reduce nesting; just do `if narrow is None: return narrow` at the start.

Reply:
oh ya, right. updated!